### ISSUE TITLE
Question: Share Lovelace Card?

### ISSUE THREAD

**Issue by mod242** [borski/ha-lucidmotors#20]
First of all: Thank you so much for this Integration. It's working great and does everything I can think of in integrating my Lucid to HA. 

Now for my question: Would you be willing to share your Lovelace-Card Code that you used for the Sample-Screenshots? This looks like exactly what I want to visualize in my HA, but I am having a hard time figuring out how to do so. Would appreciate if you'd be willing to share, how you did it.

**Comment by robertw** [comment]
I agree and thank you Yes if you could share that would be terrific thank you

**Comment by nshp** [comment]
We hear you and plan to make an easier-to-install Lovelace card at some point. For now, I posted some instructions for setting up mine here: https://lucidowners.com/threads/home-assistant-automations.7897/page-4#post-190056 - it's just annoyingly complicated right now.

**Comment by mod242** [comment]
@nshp Thank you so much! That was exactly what I was looking for. This helps me a lot. Thanks for sharing with the community
